### DISCUSSION TITLE
Controlling MOC's With Xbox

### DISCUSSION THREAD

**Discussion by RagingBullLego** [pybricks/support#1506] [Q&A]
Hi, I have a MOC I made, and it would be great if I could control it using my Xbox controller. Unfortunately, I know absolutely nothing about coding or python, so I really need help. The layout is quite simple, two drive motors connected to ports A and B, steering on C, and an auxiliary motor on D. I managed to get a far as drive and steering from the Audi example project, but I have no clue how to code the fourth motor. Help would be very much appreciated!

**Comment by laurensvalk** [comment]
Have you considered trying block coding? This can be a quick way to get to some results and preview the generated Python code as you build.

You could place an extra ‘start’ block on the canvas, and add the actions for the extra motor there.

**Reply by RagingBullLego** [reply]
Thanks, but I'd rather not pay, if I have to I can use BrickController.

**Reply by laurensvalk** [reply]
You don't have to; it can just be a helpful tool to get creative with your MOCs, and learn coding gradually.

We'll be adding more Python examples over time. If you prefer to skip the blocks, you could perhaps start with a smaller Python program and learn as you go along.
